scss/_tables.scss

```diff
@@ -4,63 +4,66 @@
   thead {
     // normal header row
     th {
-      border-bottom: solid .125rem theme-color('primary');
       border-top: none;
+      border-bottom: solid .125rem theme-color("primary");
 
       > .material-icons {
-        color: $gray-500;
         margin-top: -.5rem;
+        color: $gray-500;
       }
     }
 
     // filter row
-    tr {
-      &.column-filters {
-        background: $gray-200;
-        th {
-          border-bottom: none;
-          vertical-align: top;
-          padding: {
-            top: 1rem;
-            bottom: 1rem;
-          }
+    .column-filters {
+      background: $gray-200;
+      th {
+        vertical-align: top;
+        border-bottom: none;
+        padding: {
+          top: 1rem;
+          bottom: 1rem;
         }
       }
     }
+  }
 
-    // remove border from tables with filter row
-    &.with-filters + tbody > tr:first-of-type {
-      td, th {
+  // remove border from tables with filter row
+  /* stylelint-disable */
+  .with-filters + tbody {
+    > tr:first-of-type {
+      td,
+      th {
         border-top: none;
       }
     }
   }
+  /* stylelint-enable */
 
   td,
   th,
   tr {
     vertical-align: middle;
+  }
 
-    &.form-group {
-      text-align: center;
+  .form-group {
```

Comment:
And again here it is not in the `tr` block so it matches any tag with this class

Reply:
If we follow the bootstrap guideline we can't select something with a tag and a class/id/props at the same time

See https://stylelint.io/user-guide/rules/selector-no-qualifying-type